docs/dev/cli/virtual-usb/delete-session.md

```diff
@@ -18,7 +18,13 @@ __Description__: your Sauce Labs username.
 __Description__: your Sauce Labs access key for authentication.
 
 ```java title="Basic Example (required flags only)"
-java -jar virtual-usb-client.jar deleteSession --sessionId 12345 --username john.smith --accessKey ab015c1e-xxxx-xxxx-xxxx-xxxxxxxxxxx
+java -jar virtual-usb-client.jar deleteSession --sessionId d03a1b81-158d-4bb4-bcc9-074e43dd8465 --username john.smith --accessKey ab015c1e-xxxx-xxxx-xxxx-xxxxxxxxxxx
```

Comment:
Hi @kimsauce 

Some nitpicking now, sorry. You updated the `connect` example with optional flags with the same session id and this one isn't updated, it's still `12345` for the example with the optional flags

Reply:
don't apologize -- thank you so much, good catch!